groovy/postInit/chemistry/ChemistryOverhaul.groovy

```diff
@@ -3969,3 +3969,23 @@ CSTR.recipeBuilder()
     .duration(10)
     .EUt(140)
     .buildAndRegister()
+
+BR.recipeBuilder()
+    .fluidInputs(fluid('hydrogen_peroxide_solution') * 3000)
+    .fluidInputs(fluid('acetone') * 3000)
+    .fluidInputs(fluid('hydrochloric_acid') * 1500)
+    .fluidOutputs(fluid('diluted_hydrochloric_acid') * 3000)
+    .outputs(metaitem('dustTriacetonePeroxide') * 33)
+    .duration(20)
+    .EUt(30)
+    .buildAndRegister()
+
+BR.recipeBuilder()
+    .fluidInputs(fluid('sulfuric_acid') * 2000)
+    .fluidInputs(fluid('acetone') * 2000)
+    .fluidInputs(fluid('hydrogen_peroxide_solution') * 2000)
+    .outputs(metaitem('dustDiacetonePeroxide') * 22)
```

Comment:
make these outputs from BR recipes 1 dust only, we make exceptions for the 1 mole = n dust for some organic molecules.